cmd/broker/main.go

```diff
@@ -228,10 +227,24 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// set default formatted
+	logrus.SetFormatter(&logrus.JSONFormatter{
+		TimestampFormat: time.RFC3339Nano,
+	})
+	logs := logrus.New()
+	logs.SetFormatter(&logrus.JSONFormatter{
```

Comment:
Code duplication. Why this is necessary to repeat the sentence. If we SetFormatter for `logrus` and then create new `log` would it not be set to json already?

Reply:
In the implementation there is one global instance of logrus that we need to configure for all the places where explicit logrus instance has not been created. Additionally, `logrus.New` does not seem to copy configuration from that global instance when creating a new one. That is why duplicated configuration is required.